tools/component-extractor/build.sh

```diff
@@ -49,26 +60,46 @@ rm -rf ./material
 cp -R ~/.mui-clone/packages/mui-material/src ./material
 
 cp -R ./core/generateUtilityClass ./material/
```

Comment:
I believe these could be scripted to find any `ts` or `tsx` files in the source tree and copy them over or build them with esbuild